src/header.tsx

```diff
@@ -56,6 +56,7 @@ export const FilesCardHeader = ({ currentFilter, onFilterChange, isGrid, setIsGr
                 <SearchInput
                   placeholder={_("Filter directory")} value={currentFilter}
                   onChange={onFilterChange}
+                  onClear={event => onFilterChange(event as React.FormEvent<HTMLInputElement>, "")}
```

Comment:
This added line is not executed by any test.

Reply:
We can, will do.

Reply:
Added.